php/FreeChatHttpCode.php

```diff
@@ -0,0 +1,128 @@
+<?php
+class FreeChatHttpCode {
+    public static function http_response_code($code, $message) {
```

Comment:
every one know about http code, so you don't need map the code and message again